src/backend/gl/src/queue.rs

```diff
@@ -445,6 +444,10 @@ impl CommandQueue {
 
                 self.share.context.ClearBufferfi(target, 0, depth, stencil as _);
             }
+            com::Command::ClearTexture(color) => unsafe {
+                self.share.context.ClearColor(color[0], color[1], color[2], color[3]);
```

Comment:
this isn't clearing the currently bound texture, it's clearing the framebuffer

Reply:
Ah, I see. I got no clue why it still worked, I guess something else might have bound the texture to the framebuffer, I guess. 

Replaced with `unimplemented!()`